worker-markup-core/src/test/resources/xml/FalsePositiveCondensedHeaderMultiLangMappingExpected.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+
+    Copyright 2015-2018 Micro Focus or one of its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<root>
+    <CONTENT>
+        <email>On a Friday night, December 15 2000, Congress passed complex and highly technical legislation which essentially ensured that much of the $94,000bn over-the-counter derivatives industry would operate outside the nation's commodities laws.
```

Comment:
If this is the output then I'm glad I kept pressing for it because it would suggest that this is not working.  It is identical to the input.  Look at how the other expectations are different from the inputs.

Reply:
added the element tags, they were being returned but not used in the verification. Added them to the expectation and now assert on an empty header block.